src/engine/song.h

```diff
@@ -124,7 +124,8 @@ enum DivSystem {
   DIV_SYSTEM_YM2610_CSM,
   DIV_SYSTEM_YM2610B_CSM,
   DIV_SYSTEM_YM2203_CSM,
-  DIV_SYSTEM_YM2608_CSM
+  DIV_SYSTEM_YM2608_CSM,
+  DIV_SYSTEM_MAX // boundary for max system number
```

Comment:
Remove?

Reply:
That's just for boundary check if needed.